lib/airbrake/passenger.rb

```diff
@@ -0,0 +1,62 @@
+require 'set'
+require 'pathname'
+
+module Airbrake
+  # Integration for Phusion Passenger
+  module Passenger
+    IntegrationException = Class.new(::Exception)
```

Comment:
This was one of those things I was coding quickly and not really looking to simplify code. I also did this for a particular purpose. `IntegrationExceptions` in `Airbrake::Passenger` are exceptions that I want to `raise` to cancel the integration but then be caught and issued as a warning for the logs so as not to kill the server booting. 

In any case, I'm sure there's a neater way to do this in keeping with your current practices.